src/main/scala/org/citegraph/loading/DataFrameLoader.scala

```diff
@@ -71,7 +70,7 @@ class DataFrameLoader(val dataDirectory: String, val sparkSession: SparkSession)
         Row(rawPublishedId.toInt, year) // Output Row[Integer, Integer]
       })
 
-    sparkSession.createDataFrame(publishedDatesRDD, publishedDatesSchema)
+    sparkSession.createDataFrame(publishedDatesRDD, publishedDatesSchema).na.drop().dropDuplicates("id")
```

Comment:
Let's do this for now, but after you watch the recitation, let's discuss it again. I think he said to include nulls and duplicates for question 2.